src/binary-reader-linker.cc

```diff
@@ -243,11 +261,16 @@ Result BinaryReaderLinker::OnExport(Index index,
                                     ExternalKind kind,
                                     Index item_index,
                                     StringSlice name) {
+  if (kind == ExternalKind::Memory) {
+    WABT_FATAL("Linker does not support exported memories");
+  }
+
```

Comment:
btw I clarified my noobness mostly so you all understand I'll likely ask dumb questions and suggest wrong things, but I'm a quick learner so don't right me off 😉 . My experience with the wasm community, while small, has been great so thank you for being welcoming and understanding.